src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeSubscriptionTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="AkkaPersistenceLivenessProbeSubscriptionTest.cs" company="Petabridge, LLC">
+//      Copyright (C) 2015 - 2019 Petabridge, LLC <https://petabridge.com>
+// </copyright>
+// -----------------------------------------------------------------------
+using Akka.Actor;
+using Akka.HealthCheck.Liveness;
+using FluentAssertions;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.HealthCheck.Persistence.Tests
+{
+    public class AkkaPersistenceLivenessProbeSubscriptionTest : TestKit.Xunit.TestKit
+    {
+
+        public AkkaPersistenceLivenessProbeSubscriptionTest(ITestOutputHelper helper)
+            : base(output: helper)
+        {
+
+        }
+
+        public static string config = @"akka.persistence {
+                                         journal {
+                                                    plugin = ""akka.persistence.journal.sqlite""
+                                                    sqlite {
+                                                            class = ""Akka.Persistence.Sqlite.Journal.SqliteJournal, Akka.Persistence.Sqlite""
+                                                            auto-initialize = on
+                                                            connection-string = """ + "Filename=file:memdb.db;Mode=Memory;Cache=Shared" + @"""
```

Comment:
Technically, don't need to concatenate a string here - since the connection string itself is also a string.